components/button/Button.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import { VrButton, Text } from 'react-vr';
+
+class CnButton extends Component {
+
+  render() {
+    const { buttonStyles, textStyles } = styles;
+
+    return (
+      <VrButton
+        style={
+          [buttonStyles,
+          { backgroundColor: this.props.color ? this.props.color : 'blue',
+            height: this.props.shape === 'circle' ? 1.1 : 0.4,
+            borderRadius: this.props.shape === 'circle' ? 0.5 : 0 }]}
+      >
+        {this.props.children}
+        <Text style={textStyles}>Buy</Text>
+      </VrButton>
+    );
+  }
+}
+
```

Comment:
We can try using SyleSheets in the future. At least that's what I've seen in most of projects